services/api/kalvi/api/urls/__init__.py

```diff
@@ -12,4 +14,24 @@
         SignInEndPoint.as_view(),
         name="kalvi-sign-in",
     ),
+    path(
+        "profile/",
+        UserProfileView.as_view(),
+        name="UserDetail",
+    ),
+    path(
+        "changepassword/",
```

Comment:
could we change to`reset-password/` or `update-password/ ` ?

Reply:
did that